### ISSUE TITLE
Propagation of package updates

### ISSUE THREAD

**Issue by rob-p** [bioconda/bioconda-recipes#9942]
Hi,

  I was just curious how long it generally takes for an updated package to have the new version appear on the [anaconda cloud](https://anaconda.org/bioconda/salmon).  The relevant update was merged into master on Sunday night, but hasn't appeared yet; is that normal?

Thanks!
Rob

**Comment by bgruening** [comment]
No, I found an error it was not building. Give me a minute to create a PR to demonstrate it.

**Comment by rob-p** [comment]
ahh; that explains things!  Thanks for the ultra-fast response.

**Comment by Alanamosse** [comment]
Hi,
I had a similar issue : tests passed, PR merged, yet no anaconda update (r-rebird recipe in my case).

Same as for my recipe we can see the error "Cannot retrieve the latest commit at this time. " at https://github.com/bioconda/bioconda-recipes/tree/master/recipes/salmon

**Comment by nickloman** [comment]
I'm seeing this over at phyml too

**Comment by chadlaing** [comment]
Hi @bgruening ,

I am experiencing the same issue as @Alanamosse, my recipe (ectyper) passed all linting / tests, and was successfully merged; however, the new version (v0.8.0) has not shown up in anaconda search.

Thanks in advance for any help.
